_includes/releases/v23.1/v23.1.0-alpha.9.md

```diff
@@ -0,0 +1,80 @@
+## v23.1.0-alpha.9
+
+Release Date: April 4, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-0-alpha-9-backward-incompatible-changes">Backward-incompatible changes</h3>
+
+- The output of the [`SHOW RANGES`](show-ranges.html) command for the `crdb_internal.ranges` and `crdb_internal.ranges_no_leases` tables has been updated, and the previous output is deprecated. To enable the new command output, set the `sql.show_ranges_deprecated_behavior.enabled` [cluster setting](cluster-settings.html) to `false`. The new output will become default in v23.2. [#99618][#99618]
+
+<h3 id="v23-1-0-alpha-9-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- The [`CREATE CHANGEFEED`](create-changefeed.html) now allows you to limit the nodes that can execute a changefeed by including a locality filter in the `WITH` clause of the `CREATE CHANGEFEED` command. A node can execute the changefeed only if it was started started with a matching [`--locality`](cockroach-start#locality) flag. Replace `{locality}` with a comma-separated list of key-value pairs. [#99935][#99935]
+
+<h3 id="v23-1-0-alpha-9-sql-language-changes">SQL language changes</h3>
+
+- The new `prepared_statements_cache_size` [cluster setting](cluster-settings.html) helps to prevent [prepared statement](savepoint.html#savepoints-and-prepared-statements) leaks by automatically deallocating the least-recently-used prepared statements when the cache reaches a given size. [#99254][#99254]
+
+<h3 id="v23-1-0-alpha-9-operational-changes">Operational changes</h3>
+
+- The new `COCKROACH_DISABLE_NODE_AND_TENANT_METRIC_LABELS` [environment variable](cockroach-commands.html#environment-variables) allows you to suppress metrics from a cluster's [Prometheus endpoint](monitoring-and-alerting.html#the-prometheus-endpoint) if they conflict with labels that are applied by external tools that collect metrics from the endpoint. Set the environment variable to a comma-separated list of key-value pairs. [#99820][#99820]
+
+<h3 id="v23-1-0-alpha-9-db-console-changes">DB Console changes</h3>
+
+- The [Index Details](ui-databases-page.html#index-details) now displays the list of most-frequently-used index fingerprints to all users, rather than only to admin users, because the page now queries a view rather than a system table directly. [#99485][#99485]
```

Comment:
```suggestion
- The [**Index Details**](ui-databases-page.html#index-details) section of the **Databases** page now displays the list of most-frequently-used index fingerprints to all users, rather than only to `admin` users, because the page now queries a view rather than a system table directly. [#99485][#99485]
```